build/run_runner.sh

```diff
@@ -0,0 +1,16 @@
+HALIDE=~/ops/codegen_experiments/halide/Halide/
```

Comment:
maybe add `bash` shebang?

Reply:
This script is for fast local debug, I probably should remove it from this branch.

P.s. I even did not commit prerequisites for this script, like model. So essentially it is useless now